src/bp/ui-shared/src/Contents/Components/Form/index.tsx

```diff
@@ -176,27 +188,44 @@ const Form: FC<FormProps> = ({ bp, overrideFields, contentType, formData, fields
             <AddButton
               text={lang(field.group?.addLabel)}
               onClick={() =>
-                dispatch({ type: 'add', data: { field: field.key, renderType: field.renderType, parent } })
+                dispatch({
+                  type: 'add',
+                  data: { field: field.key, renderType: field.renderType, parent, getEmptyData }
+                })
               }
             />
           </Fragment>
         )
       case 'select':
-        const value = data[field.key] || field.defaultValue || field.options[0]?.value
-        const currentOption = field.options.find(option => option.value === value)
-
         return (
           <FieldWrapper key={field.key} label={printLabel(field, data[field.key])}>
             <Select
-              options={field.options.map(option => ({ ...option, label: lang(option.label) }))}
-              value={value}
+              bp={bp}
```

Comment:
Not needed

Reply:
Same as was I said here https://github.com/botpress/botpress/pull/3501#discussion_r445623162